src/router/kromo/dd-wrt/elegant/fresh-dark.css

```diff
@@ -176,15 +160,6 @@ form[name="wds"] select {
   background-color: #666;
   border-color: #999;
 }
-.off {
-  color: #666;
-  background-color: #1a1a1a;
-  border-color: #111;
-}
-fieldset.off {
-  border-color: #333;
-  opacity: .7;
-}
 .odd {
```

Comment:
this is now in common.css no duplication = smaller files and also dont need the extra `fieldset.off` rule